src/main/kotlin/com/gatecheck/gatecheck/api/RequestController.kt

```diff
@@ -0,0 +1,34 @@
+package com.gatecheck.gatecheck.api
+
+//import com.gatecheck.gatecheck.api.template.DefaultUserResponse
+import com.gatecheck.gatecheck.model.Request
+import com.gatecheck.gatecheck.service.RequestService
+import org.jetbrains.annotations.NotNull
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+@RestController
+@RequestMapping("/api/v1/request")
+class RequestController @Autowired constructor(private val requestService:RequestService, private val userService: UserService){
+    @GetMapping
+    fun GetRequests(@RequestParam(required=false) messages:Boolean=false,
```

Comment:
Big one here, this isn't C# hahahaha method and function names begin with a lowercase letter
Also here is an example of how the messages parameter should be written "messages: Boolean = false" notice the space between the = for the default argument and the space between the type and name